prover/src/trace/trace_lde/default/mod.rs

```diff
@@ -234,30 +230,35 @@ where
     H: ElementHasher<BaseField = E::BaseField>,
 {
     // extend the execution trace
-    #[cfg(feature = "std")]
-    let now = Instant::now();
-    let trace_polys = trace.interpolate_columns();
-    let trace_lde = RowMatrix::evaluate_polys_over::<DEFAULT_SEGMENT_WIDTH>(&trace_polys, domain);
-    #[cfg(feature = "std")]
-    debug!(
-        "Extended execution trace of {} columns from 2^{} to 2^{} steps ({}x blowup) in {} ms",
-        trace_lde.num_cols(),
-        trace_polys.num_rows().ilog2(),
-        trace_lde.num_rows().ilog2(),
-        domain.trace_to_lde_blowup(),
-        now.elapsed().as_millis()
-    );
+    let (trace_lde, trace_polys) = info_span!(
+        "Extended execution trace",
+        columns_number = field::Empty,
+        from = field::Empty,
+        to = field::Empty,
+        blowup = field::Empty
+    )
+    .in_scope(|| {
+        let trace_polys = trace.interpolate_columns();
+        let trace_lde =
+            RowMatrix::evaluate_polys_over::<DEFAULT_SEGMENT_WIDTH>(&trace_polys, domain);
+
+        tracing::Span::current().record("columns_number", &format!("{}", trace_lde.num_cols()));
+        tracing::Span::current().record("from", &format!("2^{}", trace_polys.num_rows().ilog2()));
+        tracing::Span::current().record("to", &format!("2^{}", trace_lde.num_rows().ilog2()));
+        tracing::Span::current().record("blowup", &format!("{}x", domain.trace_to_lde_blowup()));
+
+        (trace_lde, trace_polys)
+    });
 
     // build trace commitment
-    #[cfg(feature = "std")]
-    let now = Instant::now();
-    let trace_tree = trace_lde.commit_to_rows();
-    #[cfg(feature = "std")]
-    debug!(
-        "Computed execution trace commitment (Merkle tree of depth {}) in {} ms",
-        trace_tree.depth(),
-        now.elapsed().as_millis()
-    );
+    let trace_tree =
+        info_span!("Computed execution trace commitment", merkle_tree_depth = field::Empty)
```

Comment:
Let's make the following changes here:

1. Change the name to "compute_execution_trace_commitment".
2. Use `lde_domain_size.ilog2()` for `depth` field.